Automatic Validation ended with:
> 2025-02-28T23:39:43.4312606Z ##[error] The pull request contains more than one manifest. Either singleton or multi file manifest under one version directory.
 2025-02-28T23:39:43.4325176Z ##[error] The pull request contains more than one manifest. Either singleton or multi file manifest under one version directory.
 

(Automated response - build 986.)

--- a/manifests/a/Avanquestpdfforge/PDFCreator-Server/3.3.2/Avanquestpdfforge.PDFCreator-Server.installer.yaml
+++ b/manifests/a/Avanquestpdfforge/PDFCreator-Server/3.3.2/Avanquestpdfforge.PDFCreator-Server.installer.yaml
@@ -1,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Avanquestpdfforge.PDFCreator-Server
+PackageVersion: 3.3.2
+InstallerType: burn
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Silent: /COMPONENTS="MAINPROGRAM" /SILENT /DEFERLICENSECHECK
+  SilentWithProgress: /COMPONENTS="MAINPROGRAM" /SILENT /DEFERLICENSECHECK
+ElevationRequirement: elevatesSelf
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.pdfforge.org/download/pdfcreator-server/3.2.2/PDFCreator-Server-3_3_2-Setup.exe?file=PDFCreator-Server-3_3_2-Setup.exe&download
+  InstallerSha256: 400a5ef95e3d0b312edbd6e15fbc9a1e64f856ef25ecd80f28ad892ac3a8357b
+- Architecture: x86
+  InstallerUrl: https://download.pdfforge.org/download/pdfcreator-server/3.2.2/PDFCreator-Server-3_3_2-Setup.exe?file=PDFCreator-Server-3_3_2-Setup.exe&download
+  InstallerSha256: 400a5ef95e3d0b312edbd6e15fbc9a1e64f856ef25ecd80f28ad892ac3a8357b
+ManifestType: installer
+ManifestVersion: 1.5.0

--- a/manifests/a/Avanquestpdfforge/PDFCreator-Server/3.3.2/Avanquestpdfforge.PDFCreator-Server.locale.en-US.yaml
+++ b/manifests/a/Avanquestpdfforge/PDFCreator-Server/3.3.2/Avanquestpdfforge.PDFCreator-Server.locale.en-US.yaml
@@ -1,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Avanquestpdfforge.PDFCreator-Server
+PackageVersion: 3.3.2
+PackageLocale: en-US
+PackageName: PDFCreator Server
+PackageUrl: https://www.pdfforge.org/pdfcreator/editions/pdfcreator-server
+Publisher: Avanquest pdfforge GmbH
+PublisherSupportUrl: https://www.pdfforge.org/pdfcreator/support
+License: EULA
+LicenseUrl: https://docs.pdfforge.org/pdfcreator/en/pdfcreator/license/pdfcreator-eula/
+Copyright: Copyright (c) 2025 Avanquest pdfforge GmbH. All rights reserved.
+CopyrightUrl: https://docs.pdfforge.org/pdfcreator/en/pdfcreator/license/pdfcreator-eula/
+ShortDescription: PDFCreator Server works as a Windows service application and allows central management of settings and users in PDFCreator.
+Moniker: PDFCreator-Server
+Tags:
+- pdf
+- pdfcreator
+- server
+- automation
+- tool
+- utility
+- file-compression
+- file-conversion
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0

--- a/manifests/a/Avanquestpdfforge/PDFCreator-Server/3.3.2/Avanquestpdfforge.PDFCreator-Server.yaml
+++ b/manifests/a/Avanquestpdfforge/PDFCreator-Server/3.3.2/Avanquestpdfforge.PDFCreator-Server.yaml
@@ -1,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Avanquestpdfforge.PDFCreator-Server
+PackageVersion: 3.3.2
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0

